MaterialDesignThemes.Wpf/SnackbarMessageQueue.cs

```diff
@@ -352,6 +352,11 @@ private async Task ShowAsync(Snackbar snackbar, SnackbarMessageQueueItem message
             //this for now...at least it is prevent extra call back hell
             await Task.Delay(snackbar.DeactivateStoryboardDuration);
 
+            //this prevents missing resource warnings after the message is removed from the Snackbar
+            //see https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/2040
+            if (snackbar.Message != null)
+                snackbar.Message.Resources = SnackbarMessage.defaultResources;
```

Comment:
Sorry, but it should be done in `PropertyChangedCallback` of `Snackbar.MessageProperty` instead of here.

Reply:
No problem.

I made that change in [this commit](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/2051/commits/039d49c5d8cdff629dc498eb2d39b16bac69dd8a) and verified that this also works by adding this commit to my [testing branch](https://github.com/bender2k14/MaterialDesignInXamlToolkit/tree/testing/2040_default_SnackbarMessage_resources) and running the project that I added in that branch.

Since I think I made the requested change, I will mark this conversation as resolved for convenience.  Please reopen though if there is anything else to discuss.